.github/workflows/release.yml

```diff
@@ -40,6 +40,42 @@ jobs:
             --no-git-commit \
             skip
 
+  npm_publish:
+    name: Publish crates
```

Comment:
thought: Should the name of job be changed to: Publish boa_wasm.

Reply:
Fixed :)